pyupgrade/_plugins/generator_expressions_pep289.py

```diff
@@ -0,0 +1,66 @@
+import ast
+import functools
+from typing import Iterable
+from typing import List
+from typing import Tuple
+
+from tokenize_rt import Offset
+from tokenize_rt import Token
+
+from pyupgrade._ast_helpers import ast_to_offset
+from pyupgrade._data import register
+from pyupgrade._data import State
+from pyupgrade._data import TokenFunc
+from pyupgrade._token_helpers import find_closing_bracket
+from pyupgrade._token_helpers import find_comprehension_opening_bracket
+
+
+ALLOWED_FUNCS = frozenset(
+    (
+        'all',
+        'any',
+        'bytearray',
+        'bytes',
+        'frozenset',
+        'list',
+        'max',
+        'min',
+        'print',
```

Comment:
print of a list is different than print of a generator

Reply:
Indeed, thanks - the "test" I wrote to make this list only checked the return types 😳 🤦